pywhy_graphs/algorithms/cpdag.py

```diff
@@ -0,0 +1,278 @@
+from enum import Enum
+
+import networkx as nx
+
+import pywhy_graphs as pg
+
+__all__ = ["pdag_to_dag", "dag_to_cpdag", "pdag_to_cpdag", "order_edges", "label_edges"]
+
+
+class EDGELABELS(Enum):
+    """Edge labels for a CPDAG."""
+
+    COMPELLED = "compelled"
+    REVERSIBLE = "reversible"
+    UNKNOWN = "unknown"
+
+
+def is_clique(G, nodelist):
+    H = G.subgraph(nodelist)
+    n = len(nodelist)
+    return H.size() == n * (n - 1) / 2
+
+
+def order_edges(G: nx.DiGraph):
+    """Find total ordering of the edges of DAG G.
+
+    A total ordering is a topological sorting of the nodes, and then
+    ordering all possible edges according to Algorithm 4 in
```

Comment:
Might be useful to say a little more about how Alg 4 works (seems like it's using the topological sort to break ties)

Reply:
Done.